src/domain/orders/swaps/index.js

```diff
@@ -135,7 +135,7 @@ const SwapIndex = ({}) => {
             <Spinner dark />
           </Box>
         </Flex>
-      ) : !swaps.length ? (
+      ) : swaps === undefined ? (
```

Comment:
To follow the convention used in other domains, can we do `!swaps?.length` instead? :)

Reply:
Of course, thank you!

Reply:
It is ready:)